telegram_writer.py

```diff
@@ -11,12 +16,17 @@ def run(self, bots, wait_seconds):
 
         while True:
             for bot in bots:
+                if settings.DEBUG:
+                    print('Checking for news {}'.format(datetime.datetime.utcnow()))
+                chat_id = self.chat
                 try:
                     posts = bot.get()
                     for post in posts:
-                        self.telegram.sendMessage(chat_id=self.chat, text=post['text'])
+                        self.telegram.sendMessage(chat_id=chat_id, text=post['text'])
                         if post['image']:
-                            self.telegram.sendPhoto(chat_id=self.chat, photo=post['image'])
+                            self.telegram.sendPhoto(chat_id=chat_id, photo=post['image'])
                 except Exception as e:
-                    print(e)
+                    error_text = traceback.print_exc()
+                    print(error_text)
+                    self.telegram.sendMessage(chat_id=chat_id, text=error_text)
```

Comment:
Sending errors to the same group can make the group unreadable. Would be nice to specify in settings where we want to receive the errors (another specific group for errors, by email...)
The idea of having the try except in top level is to prevent the bot stop running. What about if we have an error in sendMessage, in the except part? (connectivity errors....)
We should avoid any error probability in the except part (maybe other try except just in case?)